app/controllers/users/delete_controller.rb

```diff
@@ -52,13 +53,22 @@ def send_push_notifications
       PushNotification::HttpPush.deliver(event)
     end
 
-    # rubocop:disable IdentityIdp/MailLaterLinter
     def notify_user_via_email_of_deletion
       current_user.confirmed_email_addresses.each do |email_address|
         UserMailer.with(user: current_user, email_address: email_address).
-          account_delete_submitted.deliver_now
+          account_delete_submitted.deliver_now_or_later
```

Comment:
I don't think we can / should change this. We've had issues in the past with emails delivered asynchronously in response to deletion actions, since the delivery may only occur _after_ the user is already deleted, and the email will attempt to reference values associated with the (as-of-then-deleted) user, resulting in 500 errors.

Reply:
Related:

- https://gsa-tts.slack.com/archives/C0NGESUN5/p1668538864928759
- #7495

Reply:
It'd be good if we could have some test coverage for this, but maybe it would suffice to include an inline code comment, since this would be easy to overlook in the future.